### DISCUSSION TITLE
Dsable the API Routes Response Size Limited warning

### DISCUSSION THREAD

**Discussion by italodeandra** [vercel/next.js#36011] [Help] [answered]
Hey guys.

Is it possible to disable the warning for https://nextjs.org/docs/messages/api-routes-body-size-limit ?

 I am self-hosting my Next.js app and I don't care about the size limit and I also disabled the `bodyParser` from the API (so I can't use `bodyParser: { sizeLimit: "5mb" }` because I already have the `bodyParser: false`).

**Comment by ijjk** [comment, marked as answer]
Hi, it sounds like you are looking for the below config, which is documented here https://nextjs.org/docs/api-routes/api-middlewares#custom-config 

```js
export const config = {
  api: {
    responseLimit: '8mb',
  },
}
```

**Reply by winner106** [reply]
but where to write the code?

**Reply by italodeandra** [reply]
@winner106 On the same file you have the API handler.

**Reply by jurgen-siegel** [reply]
when i put this in my api route file i get an error on vercel:

Error: Page config in /vercel/path0/app/api/upload/route.js is deprecated. Replace `export const config=…` with the following:
